cpplint_unittest.py

```diff
@@ -57,6 +59,26 @@ def u(x):
     def b(x):
         return codecs.latin_1_encode(x)[0]
 
+=======
+try:
+    xrange(0,1)
+except NameError:
+    xrange = range
+try:
+    unicode
+except NameError:
+    basestring = unicode = str
+try:
+    long
+except NameError:
+    long = int
+Py3k = (sys.version_info[0] == 3)
+if Py3k:
+  chrstr = bytes
+else:
+  def chrstr(l):
+    return ''.join([chr(x) for x in l])
+>>>>>>> Added support for Python3000
```

Comment:
Mostly same as for cpplint.py, but:
* I define function b(), you define   chrstr()